includes/options/class-amp-options-menu.php

```diff
@@ -423,6 +434,227 @@ private function list_template_conditional_options( $options, $parent = null ) {
+								<?php
+								if ( ! $is_suppressed ) {
+									echo '<strong>';
+								}
+								echo esc_html( $plugin['Name'] );
+								if ( ! $is_suppressed ) {
+									echo '</strong>';
+								}
+								?>
+							</label>
+							<?php if ( $is_suppressed && version_compare( $suppressed_plugins[ $plugin_slug ][ Option::SUPPRESSED_PLUGINS_LAST_VERSION ], $plugins[ $plugin_file ]['Version'], '!=' ) ) : ?>
+								<small>
+									<?php if ( $suppressed_plugins[ $plugin_slug ][ Option::SUPPRESSED_PLUGINS_LAST_VERSION ] && $plugins[ $plugin_file ]['Version'] ) : ?>
+										<?php
+										echo esc_html(
+											sprintf(
+												/* translators: %1: version at which suppressed, %2: current version */
+												__( '(Now updated to version %1$s since suppressed at %2$s.)', 'amp' ),
+												$plugins[ $plugin_file ]['Version'],
+												$suppressed_plugins[ $plugin_slug ][ Option::SUPPRESSED_PLUGINS_LAST_VERSION ]
+											)
+										);
+										?>
+									<?php else : ?>
+										<?php esc_html_e( '(Plugin updated since last suppressed.)', 'amp' ); ?>
+									<?php endif; ?>
+								</small>
+							<?php elseif ( ! $is_suppressed && ! empty( $errors_by_sources['plugin'][ $plugin_slug ] ) ) : ?>
+								<details>
+									<summary>
+										<?php
+										echo esc_html(
+											sprintf(
+												/* translators: %s is the error count */
+												_n(
+													'%s error',
+													'%s errors',
+													count( $errors_by_sources['plugin'][ $plugin_slug ] ),
+													'amp'
+												),
+												number_format_i18n( count( $errors_by_sources['plugin'][ $plugin_slug ] ) )
+											)
+										);
+										?>
+									</summary>
+									<ul>
+										<?php foreach ( $errors_by_sources['plugin'][ $plugin_slug ] as $validation_error ) : ?>
+											<?php
+											$edit_term_url = admin_url(
+												add_query_arg(
+													[
+														AMP_Validation_Error_Taxonomy::TAXONOMY_SLUG => $validation_error['term']->name,
+														'post_type' => AMP_Validated_URL_Post_Type::POST_TYPE_SLUG,
+													],
+													'edit.php'
+												)
+											)
+											?>
+											<li>
+												<a href="<?php echo esc_url( $edit_term_url ); ?>" target="_blank">
+													<?php echo wp_kses_post( AMP_Validation_Error_Taxonomy::get_error_title_from_code( $validation_error['data'] ) ); ?>
+												</a>
+											</li>
+										<?php endforeach; ?>
+									</ul>
+								</details>
+							<?php endif ?>
+						</li>
+					<?php endforeach; ?>
+				</ul>
+
+				<?php
+				$other_plugin_count = count( $plugins ) - count( $already_listed_plugins );
+				?>
+				<?php if ( $other_plugin_count > 0 ) : ?>
+					<details>
+						<summary>
+							<?php
+							echo esc_html(
+								sprintf(
+									/* translators: %s is the count of plugins */
+									_n(
```

Comment:
Let's find an alternative copy for `Other %d plugins`. When there are no plugins with validation issues `other` does not apply. Perhaps `Compliant plugins` or something like that?

Reply:
Good catch. Addressed now that the Other section is removed entirely in d3e772e.